services/worker-manager/schemas/v1/config-aws.yml

```diff
@@ -0,0 +1,16 @@
+$schema: "/schemas/common/metaschema.json#"
+title: AWS Provider Config
+description: An array of possible options for different regions
+type: array
+uniqueItems: true
+items:
+  type: object
+  description: |
+    An object to be passed in to AWS runInstances method.
+    See the documentation [here](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/EC2.html#runInstances-property)
+    You can configure any properties you need. Note, however, that we disallow passing in arbitrary IAM instance profiles.
+  properties:
+    not:
+      IamInstanceProfile:
```

Comment:
It'd be good to have a test that this value is rejected.  I've not seen "not" used that way before.